jaeger-core/src/test/java/io/jaegertracing/internal/senders/SenderResolverTest.java

```diff
@@ -30,6 +30,7 @@
 import java.nio.file.Path;
 import java.util.Scanner;
 import org.junit.After;
+import org.junit.Ignore;
```

Comment:
Is this import being used?

Reply:
Removed this and one other unused import( `import io.jaegertracing.spi.SenderFactory;`) in the same class.